utils/bundles.stanza

```diff
@@ -78,35 +74,75 @@ public pcb-bundle swd:
   pin swdio
   pin swdclk
 
-public pcb-bundle uart :
+public pcb-enum ocdb/bundles/UARTPins :
+  DTR
+  CTS
+  DCD
+  RI
+  DST
+  RTS
+
+pcb-bundle uart (pins:Tuple<ocdb/bundles/UARTPins>) :
```

Comment:
This definition of `uart` is identifical to `usart`. Do we require them to be unique?

Reply:
Yes - software support is very different